projects/plugins/inspect/packages/Async_Option/Async_Option.php

```diff
@@ -12,7 +10,7 @@ class Async_Option {
 	private $key;
 
 	/**
-	 * @var Storage
+	 * @var WP_Option
```

Comment:
Looks like all these changes from `Storage` to `WP_Option` are because the `get()` method in the interface doesn't specify a `$default` parameter? Can we just add it there instead?

Reply:
1ea067bf62